app/src/main/java/com/android/solvit/seeker/ui/review/CreateReview.kt

```diff
@@ -0,0 +1,298 @@
+import androidx.compose.ui.unit.sp
+import com.android.solvit.R
+import com.android.solvit.seeker.model.provider.ListProviderViewModel
+import com.android.solvit.seeker.ui.service.ProviderItem
+import com.android.solvit.shared.model.map.Location
+import com.android.solvit.shared.model.request.ServiceRequest
+import com.android.solvit.shared.model.request.ServiceRequestStatus
+import com.android.solvit.shared.model.request.ServiceRequestViewModel
+import com.android.solvit.shared.model.review.Review
+import com.android.solvit.shared.model.review.ReviewViewModel
+import com.android.solvit.shared.ui.navigation.NavigationActions
+import com.android.solvit.shared.ui.navigation.Route
+import com.google.android.gms.maps.model.LatLng
+import com.google.maps.android.compose.GoogleMap
+import com.google.maps.android.compose.Marker
+import com.google.maps.android.compose.rememberCameraPositionState
+import com.google.maps.android.compose.rememberMarkerState
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+@Composable
+fun CreateReviewScreen(
+    reviewViewModel: ReviewViewModel,
+    serviceRequestViewModel: ServiceRequestViewModel,
+    listProviderViewModel: ListProviderViewModel,
+    navigationActions: NavigationActions
+) {
+  val requestState = serviceRequestViewModel.selectedRequest.collectAsState()
+  if (requestState.value == null || requestState.value!!.status != ServiceRequestStatus.COMPLETED) {
+    navigationActions.goBack()
+  }
+  val ratingState = remember { mutableIntStateOf(0) }
+  var comment by remember { mutableStateOf("") }
+
+  // Lock Orientation to Portrait
+  val context = LocalContext.current
+  DisposableEffect(Unit) {
+    val activity = context as? ComponentActivity
+    activity?.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_PORTRAIT
+    onDispose { activity?.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_UNSPECIFIED }
+  }
+
+  Scaffold(topBar = { TopSection(navigationActions) }) { paddingValues ->
+    Column(
+        modifier =
+            Modifier.padding(paddingValues)
+                .fillMaxWidth()
+                .padding(16.dp)
+                .verticalScroll(rememberScrollState()),
+        verticalArrangement = Arrangement.Top,
+        horizontalAlignment = Alignment.CenterHorizontally) {
+          RequestBox(requestState, listProviderViewModel, navigationActions)
+          RatingBar(ratingState)
+          OutlinedTextField(
+              value = comment,
+              onValueChange = { comment = it },
+              modifier = Modifier.fillMaxWidth().height(160.dp).testTag("reviewComment"),
+              label = { Text("Comment") },
+              placeholder = { Text("Leave a comment") },
+              shape = RoundedCornerShape(16.dp),
+              colors =
+                  OutlinedTextFieldDefaults.colors(
+                      unfocusedContainerColor = Color.Transparent,
+                      focusedBorderColor = colorScheme.secondary,
```

Comment:
Please update the border color to black so that it matches the previous border style for consistency.